### PR TITLE
first draft of Danish locale.

### PR DESCRIPTION
**Pull request by ronan-mch**

### CODE REVIEW

File: lib/generators/templates/locales/da.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+da:
+  humanizer:
+    validation:
+      error: Venligst bevise, at du ikke er en robot
```
Comment (omegahm) [inline lib/generators/templates/locales/da.yml]: `Venligst bevis, at du ikke er en robot`

File: lib/generators/templates/locales/da.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        answers: ["11", "elleve"]
+      - question: Hvad er fem gange to?
+        answers: ["10", "ti"]
+      - question: "Indsæt det næste nummer i sekvensen: 10, 11, 12, 13, 14, .."
+        answers: ["15", "femten"]
+      - question: "Hvad er fem gange fem?"
+        answers: ["25", "fem og tyve", "femogtyve"]
+      - question: "Hvad er ti divideret med to?"
+        answers: ["5", "fem"]
+      - question: Hvad hedder dagen der kommer efter mandag?
```
Comment (omegahm) [inline lib/generators/templates/locales/da.yml]: `Hvad hedder dagen, der kommer efter mandag?`

File: lib/generators/templates/locales/da.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        answer: "tirsdag"
+      - question: Hvad er den sidste måned i året?
+        answer: "december"
+      - question: Hvor mange minutter er der i en time?
```
Comment (omegahm) [inline lib/generators/templates/locales/da.yml]: `Hvor mange minutter er der på en time?`

**Comment by ronan-mch** [comment]
wow, talk about efficiency @omegahm! Is that list of errors complete?

**Comment by omegahm** [comment]
Those were the things that I noted. Who am I to say if it's complete or not? :smirk:

File: lib/generators/templates/locales/da.yml (hunk 1 above)
Comment (majjoha) [inline lib/generators/templates/locales/da.yml]: Alternatively, `Bevis venligst, at du ikke er en robot`.

**Pull request closed without merging**